libs/datasets/vaccine_backfills.py

```diff
@@ -92,7 +92,7 @@ def backfill_vaccination_initiated(dataset: MultiRegionDataset) -> MultiRegionDa
 
     # Compute and keep only time series with at least one real value
     computed_initiated = administered - completed
-    computed_initiated = computed_initiated.dropna(axis=1, how="all")
+    computed_initiated = computed_initiated.dropna(axis=0, how="all")
```

Comment:
Out of curiosity, any reason to prefer the opaque `0` / `1` constants over the more descriptive `index` / `columns`?

Reply:
The value for the axis parameter often doesn't make sense to me (does 'index', 'all' mean the values in every index offset must be na to drop a column OR it iterates over the index checking all the columns?) so I started using something more obscure in hopes that I'd be careful. Clearly failed here.